src/FHist.jl

```diff
@@ -28,21 +30,25 @@ function _is_uniform_bins(A::AbstractRange{T}) where T<:Real
     true
 end
 
+const _default_overflow = false
+
 struct Hist1D{T<:Real,E} <: AbstractHistogram{T,1,E}
     hist::Histogram{T,1,E}
     sumw2::Array{Float64, 1}
     hlock::SpinLock
-    function Hist1D(h::Histogram{T,1,E}, sw2 = copy(h.weights)) where {T,E}
-        return new{T,E}(h, sw2, SpinLock())
+    overflow::Bool
+    function Hist1D(h::Histogram{T,1,E}, sw2 = copy(h.weights); overflow=_default_overflow) where {T,E}
```

Comment:
why no just `=false`?

Reply:
I wasn't sure what we wanted to be the default. If you agree false is best, I will fix it everywhere.

Reply:
I kinda line this now that I see it appears in many places lol. Maybe 10% more confusing when someone looked at this with `@less` or `@edit` but it's fine.

Reply:
Haha OK, I will keep it, but in the docstr I will put in `false` so that the user doesn't have to look up the source every time.